backend/inspirehep/search/facets.py

```diff
@@ -81,6 +98,10 @@ def inner(values):
     return inner
 
 
+def get_filters_without_excluded(filters, excluded_filters):
+    return {key: val for key, val in filters.items() if key not in excluded_filters}
```

Comment:
i'm just wondering, do we really want to filter out the filters? We'll have to keep them in sync and this is what we try to avoid

Reply:
Well, citations summary and citations by year are a special case where we need to filter out some of the filters. The graphs use citeable, refereed, citation count etc. to filter out the results once you click on one of the columns. But then the graph shouldn't update themselves as well. That is why those filters need to not change the citation summary

Reply:
👍 good point